webapp/src/Controller/API/JudgehostController.php

```diff
@@ -952,7 +951,7 @@ private function addSingleJudgingRun(
 
         if (array_key_exists($runResult, $resultsRemap)) {
             $this->logger->info('JudgeTask %d remapping result %s -> %s',
-                                [ $judgeTaskId, $runResult, $resultsRemap[$runResult] ]);
+                                [$judgeTaskId, $runResult, $resultsRemap[$runResult]]);
```

Comment:
I think that here the extra spaces improve readability, so I'm not sure I'm in favor of automatically removing them.

Reply:
I'm fine with leaving them.